src/shell/lisa_shell

```diff
@@ -373,6 +373,69 @@ echo
 
 
 ################################################################################
+# LISA Workloads utility functions
+################################################################################
+
+function lisa-wltest-init {
+export LISA_VENV="$LISA_HOME/tools/wltests/wa3.venv"
```

Comment:
Really not a fan of the missing indentation here...

Reply:
Me neither... but that's for consistency... we can always have a refactoring patch.